docs/operator-manual/upgrading/1.8-1.9.md

```diff
@@ -2,7 +2,7 @@
 
 ## Environment variables expansion
 
-Argo CD supports using [environment variables](https://argoproj.github.io/argo-cd/user-guide/build-environment/) in
+Argo CD supports using [environment variables](../../../user-guide/build-environment/) in
```

Comment:
I'm unsure, but I think we can just put `/user-guide/...` instead of having to use `../../../` here.

Reply:
We have to use relative URL since docs are served under sub-path (`https://argo-cd.readthedocs.io/en/stable/` instead of `https://argo-cd.readthedocs.io/`)

Reply:
I was thinking that RTD takes care of this, and adding prefixes (i.e. `/en/stable`) on demand, depending which language or version of the docs you are viewing - at least that's the behavior I see in e.g. `argocd-image-updater` docs.

Reply:
Got it. Updated